### ISSUE TITLE
Add Short URL List API Endpoint

### ISSUE THREAD

**Issue by nprail** [cydrobolt/polr#221]
I just built a mobile app for Polr but ran into an issue. The app can create and lookup URLs but it cannot list all of the short URLs. Neither can it delete short URLs. This is because there is no API endpoint for getting a list of short URLs. 

Here is a link to the apps website: https://filiosoft.net/polr-mobile

**Comment by nprail** [comment]
How is #227 coming? I'm looking forward to this update.

**Comment by DBX12** [comment]
Not sure if this is still relevant, the website of @nprail is offline.

**Comment by nprail** [comment]
@DBX12 I pretty much abandon that project a while ago because I stopped using Polr. Largely due to lack of movement on the project.

**Comment by DBX12** [comment]
Understandable. I'm interested in this project, maybe I contribute a bit more in the future.

**Comment by nprail** [comment]
@DBX12 Yeah, I had started working on an alternative open source URL shortener in Node.js but never finished it.

**Comment by PLNech** [comment]
Implemented in #647: `GET /api/v2/action/list` lists your links (admins see all) with an optional substring filter, reusing the existing `api` key middleware. Includes tests and docs. Also available now in the [suivr fork](https://github.com/PLNech/suivr) if you need it before review.
